tests/ballerina-integration-test/src/test/resources/http/httpservices/03_atm_locator_service.bal

```diff
@@ -123,21 +106,18 @@ service<http:Service> Banklocator bind serviceEnpoint {
     product (endpoint caller, http:Request req) {
         http:Response res = new;
         var jsonRequest = req.getJsonPayload();
-        match jsonRequest {
-            json bankLocator => {
-                string zipCode;
-                zipCode = extractFieldValue2(bankLocator.BranchLocator.ZipCode);
-                json payload = {};
-                if (zipCode == "95999") {
-                    payload = {"ABCBank":{"BranchCode":"123"}};
-                } else {
-                    payload = {"ABCBank":{"BranchCode":"-1"}};
-                }
-                res.setJsonPayload(payload);
-            }
-            error err => {
-                io:println("Error occurred while reading bank locator request");
+        if (jsonRequest is json) {
+            string zipCode;
```

Comment:
```suggestion
            string zipCode = extractFieldValue2(jsonRequest.BranchLocator.ZipCode);
```
Better initalize as above since we are removing default values